### ISSUE TITLE
Mute audio

### ISSUE THREAD

**Issue by tomerhatav** [shogo4405/lf.swift#54]
Hello @shogo4405 and thank you for an awesome framework.
Is there any why to mute audio while streaming? (sending silence AAC packets)

**Comment by shogo4405** [comment]
I didn't mean to mute audio.
Tell me details.

**Comment by tomerhatav** [comment]
@shogo4405 the audio is fine, I want to know if I can mute the audio manually.

**Comment by shogo4405** [comment]
@tomerhatav 
OK. I understand. No feature.
But I think it is necessary. I will try to implement.

**Comment by shogo4405** [comment]
I implemented it. https://github.com/shogo4405/lf.swift/commit/1634bdee97b971fed8f970ef1ea517c972aaaec8
```swift
rtmpStream.audioSettings = [
    "muted": true, // if you want a mute audio.
]
```
I will release 0.3.2 soon.

**Comment by tomerhatav-zent** [comment]
Thank you for your hard work @shogo4405 you are awesome!

**Comment by tomerhatav** [comment]
Thank you for your hard work @shogo4405 you are awesome!

**Comment by shogo4405** [comment]
released 0.3.2
